### PR TITLE
6 review

### PR DESCRIPTION
**Pull request by SentinelProxima**

### CODE REVIEW

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -16,8 +17,9 @@ const Dashboard = (props) => {
                 <img className="review-image" src={item.review_img_url} alt={item.title}/>
                 <p>{item.title}</p>
                 <p class="designer">{item.designer}</p>
-                <p>Review by {item.owner}</p>
-                <p class="review-text">{item.review_body}</p>
+                <Link to={`/review/${item.review_id}`}>
```
Comment (Rosem10) [inline src/components/Dashboard.js]: I'd suggest extracting out the info from line 16 into it's own component for readability!

File: src/components/SingleCategory.js [hunk 2]
```diff
@@ -18,8 +18,10 @@ const SingleCategory = (props) => {
                 <img className="review-image" src={item.review_img_url} alt={item.title}/>
                 <p>{item.title}</p>
                 <p class="designer">{item.designer}</p>
```
Comment (Rosem10) [inline src/components/SingleCategory.js]: And if you extracted that logic as suggested in my previous comment - you could re-use it here and your code would be so much DRY-er!

File: src/components/SingleReview.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const SingleCategory = (props) => {
+    const [review, setReview] = useState([]);
+    const { id } = useParams();
+    useEffect(() => {
+        fetch(`https://pacific-citadel-03998.herokuapp.com/api/reviews/${id}`)
```
Comment (Rosem10) [inline src/components/SingleReview.js]: I'd also suggest placing all of you request functions in one api.js file, and then importing in each one as you need it.

**Review by Rosem10 — APPROVED** [review]
A couple of suggestions for you, Michael - but overall, good stuff!

File: src/utils/api.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const fetchAllReviews = () => {
```
Comment (itskatherine) [inline src/utils/api.js:1]: Nice separation out of API calls 🙂

File: src/components/SingleReview.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
```
Comment (itskatherine) [inline src/components/SingleReview.js:1]: This is way better as its own component, great stuff

**Review by itskatherine — COMMENTED** [review]
Great stuff Michael, good to merge 👍

**Pull request merged**